### ISSUE TITLE
Thesis Meeting | July 22, 2024 - **11:30am** - ITB 167

### ISSUE THREAD

**Issue by samm82** [samm82/TestGen-Thesis#73]
1. Updates to graphs
   1. Coloring (the legend still needs to be updated)
   2. Ability to create/modify subsets!
   3. Ability to pull out relevant information to be build into my thesis!
2. Thoughts on my current "massaging"? See #71, #40, and #65
4. Any areas that stand out to start to analyze?

**Comment by samm82** [comment]
We mainly camped out on point 1, as well as talking about how problems/solutions can be separated. The main takeaway is that the ability to analyze these graphs automatically for discrepancies should be the focus of my paper, which seems plausible to have ready for the abstract submission!

Points 2 and 3 should wait until after that deadline 😅

**Comment by smiths** [comment]
Good summary @samm82, although the main takeaway is more nuanced than what you said.  The main takeaway for the paper is that the state of testing terminology, including the terminology in the standards, is terrible.  Your analysis of the graphs is how you are able to make a complete argument highlighting all the problems.  The tool is important, but it is the results that will make the paper impactful.

**Comment by samm82** [comment]
Good call; I guess implicit in my remark on focusing on "the ability to analyze these graphs automatically for discrepancies" was the inclusion of these analyses!

In particular, the EXPLICIT separation of problem/proposal will help keep this focus, and prevent me from getting too carried away in analysis! 😅
